### ISSUE TITLE
Error with inpainting (Kernel size can't be greater than actual input size)

### ISSUE THREAD

**Issue by Miraihi** [Acly/krita-ai-diffusion#36]
Eventually the inpanting in 100% strength ceases to work until I relaunch Krita. (Error at about 30%: `Calculated padded input size per channel: (34 x 2). Kernel size: (3 x 3). Kernel size can't be greater than actual input size`)

Client Log:
```
2023-11-07 00:20:37,158 ERROR Job b97d2c1b-cb9d-4fe6-8b8f-925e412e08ad failed: Calculated padded input size per channel: (34 x 2). Kernel size: (3 x 3). Kernel size can't be greater than actual input size
['  File "E:\\AI\\ComfyUI\\ComfyUI\\execution.py", line 152, in recursive_execute\n    output_data, output_ui = get_output_data(obj, input_data_all)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\execution.py", line 82, in get_output_data\n    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\execution.py", line 75, in map_node_over_list\n    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\nodes.py", line 1236, in sample\n    return common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\nodes.py", line 1206, in common_ksampler\n    samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative, latent_image,\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\sample.py", line 97, in sample\n    samples = sampler.sample(noise, positive_copy, negative_copy, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 785, in sample\n    return sample(self.model, noise, positive, negative, cfg, self.device, sampler(), sigmas, self.model_options, latent_image=latent_image, denoise_mask=denoise_mask, callback=callback, disable_pbar=disable_pbar, seed=seed)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 690, in sample\n    samples = sampler.sample(model_wrap, sigmas, extra_args, callback, noise, latent_image, denoise_mask, disable_pbar)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 630, in sample\n    samples = getattr(k_diffusion_sampling, "sample_{}".format(sampler_name))(model_k, noise, sigmas, extra_args=extra_args, callback=k_callback, disable=disable_pbar, **extra_options)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\utils\\_contextlib.py", line 115, in decorate_context\n    return func(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\k_diffusion\\sampling.py", line 701, in sample_dpmpp_2m_sde_gpu\n    return sample_dpmpp_2m_sde(model, x, sigmas, extra_args=extra_args, callback=callback, disable=disable, eta=eta, s_noise=s_noise, noise_sampler=noise_sampler, solver_type=solver_type)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\utils\\_contextlib.py", line 115, in decorate_context\n    return func(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\k_diffusion\\sampling.py", line 613, in sample_dpmpp_2m_sde\n    denoised = model(x, sigmas[i] * s_in, **extra_args)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 323, in forward\n    out = self.inner_model(x, sigma, cond=cond, uncond=uncond, cond_scale=cond_scale, cond_concat=cond_concat, model_options=model_options, seed=seed)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\k_diffusion\\external.py", line 125, in forward\n    eps = self.get_eps(input * c_in, self.sigma_to_t(sigma), **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\k_diffusion\\external.py", line 151, in get_eps\n    return self.inner_model.apply_model(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 311, in apply_model\n    out = sampling_function(self.inner_model.apply_model, x, timestep, uncond, cond, cond_scale, cond_concat, model_options=model_options, seed=seed)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 289, in sampling_function\n    cond, uncond = calc_cond_uncond_batch(model_function, cond, uncond, x, timestep, max_total_area, cond_concat, model_options)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\samplers.py", line 265, in calc_cond_uncond_batch\n    output = model_function(input_x, timestep_, **c).chunk(batch_chunks)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\model_base.py", line 63, in apply_model\n    return self.diffusion_model(xc, t, context=context, y=c_adm, control=control, transformer_options=transformer_options).float()\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\ldm\\modules\\diffusionmodules\\openaimodel.py", line 627, in forward\n    h = forward_timestep_embed(module, h, emb, context, transformer_options)\n', '  File "E:\\AI\\ComfyUI\\ComfyUI\\comfy\\ldm\\modules\\diffusionmodules\\openaimodel.py", line 61, in forward_timestep_embed\n    x = layer(x)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 460, in forward\n    return self._conv_forward(input, self.weight, self.bias)\n', '  File "E:\\AI\\ComfyUI\\python\\lib\\site-packages\\torch\\nn\\modules\\conv.py", line 456, in _conv_forward\n    return F.conv2d(input, weight, bias, self.stride,\n']
```

EDIT:
Sometimes even restarting doesn't help. Certainly some kind of problem with the ControlNet plugin and your new method of inpainting. Didn't encounter that kind of error before.

**Comment by Acly** [comment]
Pretty sure it's related to the area conditioning. I don't understand how restarting would help, that sounds strange. What I could find that definitely reproduces this is a high resolution image where you inpaint (100% with prompt) a relatively small area in the bottom right.

**Comment by Acly** [comment]
Hopefully fixed in [v1.6.0](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.6.0), let me know if you still run into it.
